src/libmongoc/doc/learn/get/installing.rst

```diff
@@ -30,8 +32,110 @@ management repositories of `common Linux distributions <linux_>`_ and
   For the most up-to-date versions of the C driver libraries, prefer to instead
   :doc:`build from source <from-source>`.
 
+.. seealso::
 
-.. _linux:
+  For a listing and common reference on available packages, refer to
+  :doc:`/ref/packages`.
+
+
+.. index::
+  package managers; Conan
+  package managers; vcpkg
+  pair: installation; package managers
+  :name: installing.pkgman
+
+Cross Platform Installs Using Library Package Managers
+******************************************************
+
+Various library package managers offer |libbson| and |libmongoc| as installable
+packages, including Conan_ and vcpkg_. This section will detail how to install
+using those tools.
+
+.. _conan: https://conan.io/
+.. _vcpkg: https://vcpkg.io/
+
+
+.. index::
+  ! pair: installation; vcpkg
+
+Installing using vcpkg
+======================
+
+.. note::
+  This page will not detail how to get started using vcpkg_. For that, refer to
+  `"Get started with vcpkg" (external)`__
+
+  __ https://vcpkg.io/en/getting-started
+
+.. tab-set::
+
+  .. tab-item:: vcpkg Manifest Mode (Recommended)
+
+    In `vcpkg manifest mode`__, add the desired libraries to your project's
+    ``vcpkg.json`` manifest file:
+
+    __ https://learn.microsoft.com/en-us/vcpkg/users/manifests
+
+    .. code-block:: js
+      :caption: ``vcpkg.json``
+      :linenos:
+
+      {
+        // ...
+        "dependencies": [
+          // ...
+          "mongo-c-driver"
+        ]
+      }
+
+    When you build a CMake project with vcpkg integration and have a
+    ``vcpkg.json`` manifest file, vcpkg will automatically install the project's
+    dependencies before proceeding with the configuration phase, so no
+    additional manual work is required.
+
+
+  .. tab-item:: vcpkg Classic Mode
```

Comment:
Do we need to bother documenting and testing Classic Mode if it is not recommended either by our documentation or the vcpkg documentation?

Reply:
Until its outright deprecated, I think it is worthwhile to keep this. It isn't much to maintain here (only four paragraphs), especially as the feature is probably on the way out and won't be seeing significant modifications